web/sites/default/settings.php

```diff
@@ -53,16 +53,33 @@
   $config['search_api.server.solr']['name'] = 'Lagoon Solr - Environment: ' . getenv('LAGOON_PROJECT');
 }
 
-### Lagoon Redis connection.
-if (getenv('LAGOON')){
-  $settings['redis.connection']['interface'] = 'PhpRedis';
-  $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
-
-  $settings['cache_prefix']['default'] = getenv('LAGOON_PROJECT') . '_' . getenv('LAGOON_GIT_SAFE_BRANCH');
+// Redis configuration.
+if (getenv('LAGOON')) {
+  $redis = new \Redis();
+  $redis_host = getenv('REDIS_HOST') ?: 'redis';
+  $redis_port = getenv('REDIS_PORT') ?: 6379;
```

Comment:
why this change? this will not work:
```
[drupal-example]master@cli-drupal:/app$ env | grep REDIS_PORT
REDIS_PORT=tcp://172.30.22.201:6379
```

Reply:
Reverted. Maybe it makes sense to change the name of the `REDIS_PORT` variable since it might lead to misunderstanding.

Reply:
mhh what to you mean with change the name of the variable? the variable is auto generated by kubernetes based on the name of the service, so we cannot really change the name.

Reply:
Ok I see. I thought it was something Lagoon related.
I expect a variable called `REDIS_PORT` contains only a port number and not a full address, protocol included.
Can we merge the PR now?

Reply:
@vincenzodnp like sean said we don't have proof yet that this code actually works, I'm working on proofing this now.